js/src/product-feed/submission-success-guide/index.js

```diff
@@ -19,19 +19,14 @@ import GuidePageContent, {
 	ContentLink,
 } from '.~/components/guide-page-content';
 import AddPaidCampaignButton from '.~/components/paid-ads/add-paid-campaign-button';
-import { GUIDE_NAMES, LOCAL_STORAGE_KEYS } from '.~/constants';
+import { glaData, GUIDE_NAMES, LOCAL_STORAGE_KEYS } from '.~/constants';
```

Comment:
💅  This `glaData` could be in UPPERCASE because it's a constant. I'm not sure if this is coming from the past.

Reply:
The `glaData` references to `window.glaData`, and its declaration and its lower-case properties are created by PHP side. In addition, it's more like preloading data rather than a collection of constants. Considering this seems to be out of the scope of this PR/project, I'm not leaning towards adjusting it in this PR.